app/views/competitions/_nav.html.erb

```diff
@@ -79,6 +79,13 @@
           }
         end
         if current_user&.can_upload_competition_results?(@competition)
+          unless @competition.probably_over?
+            nav_items << {
+              text: t('.menu.newcomer_checks'),
+              path: competition_newcomer_checks_path(@competition),
+              icon: "users",
+            }
+          end
```

Comment:
With this menu item, the feature gets "launched" immediately. Would you be open to hide this feature behind some sort of test flag, or maybe some `unless Rails.env.production? && EnvConfig.WCA_LIVE_SITE?` check, to let a few more people (especially WRT) test this on staging first?

Reply:
Yeah that sounds like a good plan!

Reply:
Done